packages/@tailwindcss-standalone/scripts/build.ts

```diff
@@ -0,0 +1,71 @@
+import { $ } from 'bun'
+import { createHash } from 'node:crypto'
+import { mkdir, readFile, writeFile } from 'node:fs/promises'
+import * as path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url))
+
+async function buildForPlatform(triple: string, outfile: string) {
+  // We wrap this in a retry because occasionally the atomic rename fails for some reason
+  for (let i = 0; i < 5; ++i) {
```

Comment:
Definitely a bug in Bun

Is this happening on all platforms or on a specific platform?

Reply:
When I ran into this it was on macOS. Not sure if it applies to other platforms. It happened enough that I added a loop for it but it was **very** inconsistently reproducible.